src/utils/shuffle.ts

```diff
@@ -93,3 +202,5 @@ export const shuffle = ({ images, description }: WorkType) => {
   )
   return shuffled // Return the last shuffled array, even if invalid, to prevent infinite loops.
 }
+
+export { shuffle as shuffle }
```

Comment:
I kept the old function, just replace `export { shuffle as shuffle }` with `export { oldShuffle as shuffle }` to use old one if you wanna compare.